src/screens/FeedScreen/RenderList.js

```diff
@@ -84,6 +93,8 @@ const RenderListFeed = (props) => {
     });
   };
 
+  const {followUnfollow} = usePostHook();
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

You've added a new hook `usePostHook`. Ensure that its implementation is correct and it doesn't introduce any side effects.

```diff
+  const {followUnfollow} = usePostHook();
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->